### PR TITLE
Circle-ci fix

### PR DESCRIPTION
**Pull request by ctrl-z-9000-times**

### CODE REVIEW

**Comment by ctrl-z-9000-times** [comment]
The brew error message told me how to fix it :)

File: circle.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@ jobs:
             echo 'pip version: ' && python -m pip --version 
       - run:
           name: Installing cmake 
-          command: brew install cmake
+          command: brew install cmake || brew install cmake
```
Comment (breznak) [inline circle.yml]: glad this works, but it's a weird command :smile: 
> do something, or do that same thing again ... ?

Comment (ctrl-z-9000-times) [reply circle.yml]: It only retries the command if it doesn't work the first time.

Comment (breznak) [reply circle.yml]: right..I'd assume computers be deterministic in this matter :P

Comment (ctrl-z-9000-times) [reply circle.yml]: The brew error message told me to run it a second time.  It was a problem with updating cmake to latest version, afterwhich it didn't output some  critical file and failed.

**Pull request merged**